Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python
+
+RUN mkdir /tmp/dcos-cli
+RUN apt-get update && apt-get install -y wget curl python-pip less tmux
+
+RUN pip install virtualenv httpie
+
+ENV DCOS_CONFIG=/root/.dcos/dcos.toml
+
+ADD https://s3.amazonaws.com/downloads.mesosphere.io/dcos-cli/install.sh /tmp/
+RUN /bin/bash /tmp/install.sh /tmp/dcos-cli http://localhost || true
```

Comment:
The way I usually do it is to set the url after the fact, this value is valid for script install time.